baxter_description/urdf/convert_to_urdf_and_sdf.sh

```diff
@@ -0,0 +1,2 @@
+xacro --inorder baxter.urdf.xacro gazebo:=true > baxter.urdf
+gz sdf --print baxter.urdf > baxter.sdf
```

Comment:
This could use a `roscd` at the beginning, to ensure it's operating in the proper directory:
```
roscd baxter_description/urdf
```